mirage-net.version

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
this is not needed with beta9 which adds a new `jbuilder subst` CLI option to do exactly the same watermarking as topkg does.